make/istio/stop

```diff
@@ -21,3 +21,16 @@ fi
 kubectl get customresourcedefinitions -o=go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
   | grep "istio.io" \
   | xargs --no-run-if-empty kubectl delete customresourcedefinitions
+
+# Delete permission definitions.
+kubectl get podsecuritypolicy -o=go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
+  | grep "istio" \
+  | xargs --no-run-if-empty kubectl delete podsecuritypolicy
+
+kubectl get clusterrole -o=go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
```

Comment:
Don't need to do this here, but I just realized that all of this is easier as:
```bash
kubectl get CustomResourceDefinitions,PodSecurityPolicy,ClusterRole,ClusterRoleBinding -o name \
  | grep istio \
  | xargs --no-run-if-empty kubectl delete
```

(Of course this only works if the grep works for all of the resource types)